db/migrate/20160804103616_add_booking_reference_to_appointments.rb

```diff
@@ -0,0 +1,9 @@
+class AddBookingReferenceToAppointments < ActiveRecord::Migration
+  def change
+    add_column :appointments, :booking_ref, :string, default: '', null: false
+    add_column :appointment_versions, :booking_ref, :string, default: '', null: false
+
+    execute 'UPDATE appointments SET booking_ref = uid WHERE delivery_partner = \'tpas\''
```

Comment:
If rubocop were linting this code it would tell you to use double-quoted strings or `%(` notation rather than escaping quotes 😉 But then I'm not rubocop...